products/gerrit.md

```diff
@@ -0,0 +1,116 @@
+
+releases:
+-   releaseCycle: "3.7"
+    releaseDate: 2022-11-09
+    support: true
+    eol: false
+    latest: "3.7.0"
+    latestReleaseDate: 2022-11-09
+-   releaseCycle: "3.6"
+    releaseDate: 2022-05-23
+    support: true
+    eol: false
+    latest: "3.6.3"
+    latestReleaseDate: 2022-11-09
+-   releaseCycle: "3.5"
+    releaseDate: 2021-12-23
+    support: true
+    eol: false
+    latest: "3.5.4"
+    latestReleaseDate: 2022-11-08
+-   releaseCycle: "3.4"
+    releaseDate: 2021-5-17
+    support: 2022-11-09
+    eol: true
+    latest: "3.4.8"
+    latestReleaseDate: 2022-11-08
+-   releaseCycle: "3.3"
+    releaseDate: 2020-12-01
+    support: 2022-05-24
+    eol: true
+    latest: "3.3.9"
+    latestReleaseDate: 2021-12-23
+-   releaseCycle: "3.2"
+    releaseDate: 2020-06-01
+    support: 2021-12-07
+    eol: true
+    latest: "3.2.14"
+    latestReleaseDate: 2021-11-21
+-   releaseCycle: "3.1"
+    releaseDate: 2019-11-14
+    support: 2021-05-19
+    eol: true
+    latest: "3.1.16"
+    latestReleaseDate: 2021-08-18
+-   releaseCycle: "3.0"
+    releaseDate: 2019-05-15
+    support: 2020-12-01
+    eol: true
+    latest: "3.0.16"
+    latestReleaseDate: 2021-01-26
+-   releaseCycle: "2.16"
+    releaseDate: 2018-11-14
+    support: 2020-06-01
+    eol: true
+    latest: "2.16.28"
+    latestReleaseDate: 2021-11-19
+-   releaseCycle: "2.15"
+    releaseDate: 2018-03-28
+    support: 2019-11-15
+    eol: true
+    latest: "2.15.22"
+    latestReleaseDate: 2021-01-26
+-   releaseCycle: "2.14"
+    releaseDate: 2017-04-26
+    support: 2019-05-31
+    eol: true
+    latest: "2.14.22"
+    latestReleaseDate: 2020-11-18
+-   releaseCycle: "2.13"
+    releaseDate: 2016-09-21
+    support: false
+    eol: true
+    latest: "2.13.14"
+    latestReleaseDate: 2019-04-19
+-   releaseCycle: "pre 2.13"
+    releaseDate: 2015-12-22
+    support: false
+    eol: true
+
+---
+
+> [Gerrit Code Review](https://www.gerritcodereview.com/) is a web-based code
+> review tool built on [Git version control](https://git-scm.com/).
+> Find a quick introduction
+> [here](https://gerrit-review.googlesource.com/Documentation/intro-quick.html)
+
+- [Release Policy](https://www.gerritcodereview.com/support.html): The Gerrit
+  open-source community actively supports the last 3 releases on a best effort
+  basis. Older releases are not actively maintained but may still receive
+  important fixes (e.g. security fixes), but there is no guarantee for this.
+- End of life for old release happens implicitly when a new Gerrit version is
+  released, and is announced via the
+  [project news](https://www.gerritcodereview.com/news.html) and on the
+  [mailing list](https://groups.google.com/g/repo-discuss).
+- Gerrit v2.16 Support: Existing users having issues with the migration
+  to/through Gerrit v2.16 can still use the support on the mailing
+  list as usual and it’s possible that community members will be able
+  to assist them.
```

Comment:
- The Gerrit 2.16 support guarantee is very weak, so not worth mentioning.
- releasePolicyLink is already present in the template, so no need to link to it again.
- Switched from bullets to just 2 paras.


```suggestion
The Gerrit open-source community actively supports the last 3 releases on a best effort
basis. Older releases are not actively maintained but may still receive
important fixes (e.g. security fixes), but there is no guarantee for this.

End-of-life for old releases happens implicitly when a new Gerrit version is
released, and is announced via the [project news](https://www.gerritcodereview.com/news.html)
and on the [mailing list](https://groups.google.com/g/repo-discuss).
```

Reply:
done